### PR TITLE
Update IT requirements to include iPad iOS 13.4 w/ keyboard/mouse

### PR DESCRIPTION
**Pull request by maddiedierker**
Updating /educate/it to call out that our upper-level courses work with an iPad running iPadOS 13.4+ with a bluetooth keyboard and mouse.

| Before  | After |
| ------------- | ------------- |
| Screen Shot 2020-07-23 at 4 28 47 PM  | Screen Shot 2020-07-23 at 4 28 29 PM  |
| Screen Shot 2020-07-23 at 4 29 01 PM | Screen Shot 2020-07-23 at 4 28 18 PM  |

## Links

- [based on this Slack thread](https://codedotorg.slack.com/archives/C1B3PNDL7/p1595278385372100)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jmkulwik — APPROVED** [review]
Looks great!
Don't forget to update the IT support blog post too. https://support.code.org/hc/en-us/articles/202591743-Technical-requirements-for-Code-org
Instructions here: https://codedotorg.slack.com/archives/C039X1ZLX/p1592953086043200

File: pegasus/sites.v3/code.org/public/educate/it.md [hunk 1]
```diff
@@ -40,8 +40,11 @@ For the very best experience with all Code.org content, we recommend consulting
 | **To use Internet Simulator**   | `https://api.pusherapp.com`<br/>`wss://ws.pusherapp.com:443`                                                                                           |
 | **To use Web Lab** | `https://downloads.computinginthecore.org`<br/>`https://codeprojects.org`                                                                              |
 
-## Mobile Support Details
-All CS Discoveries, CS Principles, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad, like on MacBooks) are required. Microsoft Surface Pros with a keyboard attachment and Chromebooks are compatible. iPads or other tablets with a keyboard attachment are not compatible.
+## Mobile and Tablet Support Details
+
+All CS Discoveries, CS Principles, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad, like on MacBooks) are required. Most tablets are not supported, with the following exceptions:
+- Microsoft Surface Pro with an external keyboard
+- iPad running iOS 13.4 or later with an external keyboard and a bluetooth mouse or touchpad
```
Comment (sureshc) [inline pegasus/sites.v3/code.org/public/educate/it.md]:
```suggestion
- iPad running iPadOS 13.4 or later with an external keyboard and an external mouse or touchpad
```

**Review by sureshc — APPROVED** [review]
Yaaay! More students should be able to use more of our courses with this guidance :)

File: pegasus/sites.v3/code.org/public/educate/it.md (hunk 1 above)
Comment (sureshc) [reply pegasus/sites.v3/code.org/public/educate/it.md]: It's rare, but technically USB mice/touchpads are supported with iPads.

**Pull request merged**